test/components/popup.spec.tsx

```diff
@@ -98,28 +134,30 @@ describe('<Popup />', function() {
                 <span data-automation-id="SPAN">Popup Body</span>
             </Popup>);
 
+        const popup = new PopupTestDriver(result as Popup);
         return waitFor(() => {
-            expect(bodySelect(portalId)!.getBoundingClientRect().width)
+            expect((popup.root! as Element).getBoundingClientRect().width)
```

Comment:
don't need as Element or "!" here

Reply:
fixed after did the `HTMLElement` fix, so not sure, but now it works anyway so all is good